src/Base/PODVector.cpp

```diff
@@ -61,6 +61,9 @@ void make_PODVector(py::module &m, std::string typestr, std::string allocstr)
         .def(py::init<>())
         .def(py::init<std::size_t>(), py::arg("size"))
         .def(py::init<PODVector_type&>(), py::arg("other"))
+        .def("assign", [](PODVector_type & pv, T const & value){
+            pv.assign(pv.size(), value);
+        }, py::arg("value"))
```

Comment:
Overloaded because `vector.assign(vector.size(), 42)` to set a value is a bit cumbersome.